src/TTGamesExplorerRebirthUI/Forms/TextForm.cs

```diff
@@ -20,6 +20,14 @@ public TextForm(string fileName, byte[] fileBuffer, object datFile = null)
 
             fastColoredTextBox1.BackColor = Colors.DarkBackground;
             fastColoredTextBox1.SelectionColor = Colors.BlueHighlight;
+            
+            if (fileBuffer.Length > 10000000) // 10 MB
+            {
+                fastColoredTextBox1.Text = "No file open";
+                MessageBox.Show("Text viewer does not support files over 10MB! \n\nUse Notepad or Visual Studio instead.", "Text Viewer", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                return;
```

Comment:
```suggestion

                return;
```
Always a new empty line before a `return`.